### ISSUE TITLE
MaterialForm WindowState.Normal Issue

### ISSUE THREAD

**Issue by byungmeo** [leocb/MaterialSkin#335]
![2022-02-09 23-30-11](https://user-images.githubusercontent.com/37506142/153224146-6ce3c283-a00f-4d32-9e47-dacae2de1f32.gif)

If FormBorderStyle = FixedSingle -> WindowState.Normal Method have problem.

**Comment by valimaties** [comment]
Sorry, but I cannot reproduce.
Can you upload your test project here?

**Comment by byungmeo** [comment]
i found it! if set FormBorderStyle Attribute to FixedSingle, have same problem.
i think this is my fault.
I just set it up because the border of the screen was hard to distinguish.
But it didn't work, and there was a bug like that just now.

**Comment by valimaties** [comment]
> ... I just set it up because the border of the screen was hard to distinguish. 

Yes, indeed, I also see this as a little issue, having no border. 
I have this issue with border on modal forms, but I solved it by setting the main form's content as a blur content (I take a snapshot of form before displaying modal form), and in this situation the modal form is in a good visibility.

**Comment by byungmeo** [comment]
@valimaties 
That's a good idea. I'll try using it.

**Comment by valimaties** [comment]
I think this must be somehow repaired, so, in my opinion, no, don't close it.
Let's see what @orapps44  thinks about it. He is the one who maintain this project, so he is the one who must tell better than me 🙂

**Comment by progmars** [comment]
I have similar problem when my form grows vertically every time I restore it from minimized state. I tried different FormBorderStyle - FixedDialog, Sizable, the same thing happens.

**Comment by progmars** [comment]
Found a fix:

```
        protected override void SetBoundsCore(int x, int y, int width, int height, BoundsSpecified specified)
        {
            if (DesignMode || IsDisposed)
            {
                base.SetBoundsCore(x, y, width, height, specified);
                return;
            }

            base.SetBoundsCore(RestoreBounds.Left, RestoreBounds.Top, 
                RestoreBounds.Width, RestoreBounds.Height, BoundsSpecified.All);
        }
```
